snakes_cafe.py

```diff
@@ -1,31 +1,209 @@
-         ('Spring Rolls', 4.00): 0, ('Rings', 4.00): 0,
-         ('Shrimp bites', 4.00): 0, ('Wontons', 4.00): 0,
-         ('Crab Dip', 4.00): 0, ('Sliders', 4.00): 0,
-         ('Clams', 8.00): 0},
-         'Entrees': {('Salmon', 5.00): 0, ('Steak', 6.00): 0,
-                     ('Meat Tornado', 4.00): 0, ('A Literal Garden', 3.00): 0,
-                     ('Pasta', 4.00): 0, ('Ribs', 4.00): 0,
-                     ('Cabbage Rolls', 8.00): 0, ('Pizza', 4.00): 0,
-                     ('Paella', 4.00): 0},
-         'Desserts': {('Ice Cream', 500.0): 0, ('Cake', 30.00): 0,
-                      ('Pie', 3.00): 0, ('Pudding', 4.00): 0,
-                      ('Fruit', 4.00): 0, ('Sorbet', 4.00): 0,
-                      ('Torte', 4.00): 0, ('Flan', 4.00): 0,
-                      ('Apple', 4.00): 0},
-         'Drinks': {('Coffee', 4.00): 0, ('Tea', 3.00): 0,
-                    ('Blood of the Innocent', 6.00): 0, ('Pop', 4.00): 0,
-                    ('Wine', 4.00): 0, ('Beer', 4.00): 0,
-                    ('Sake', 4.00): 0, ('Cocoa', 4.00): 0,
-                    ('Evian', 4.00): 0},
-         'Sides': {('Frys', 4.00): 0, ('Salad', 3.00): 0,
-                   ('Bread', 6.00): 0, ('Slaw', 4.00): 0, ('Soup', 4.00): 0,
-                   ('Rice', 4.00): 0, ('Spinach', 4.00): 0,
-                   ('Sauce', 4.00): 0, ('Crab dip', 4.00): 0}}
+from uuid import uuid4
+
+
+menus = {'Appetizers': {('Wings', 2.00): 3, ('Cookies', 15.00): 3,
+         ('Spring Rolls', 4.00): 3, ('Rings', 4.00): 3,
+         ('Shrimp bites', 4.00): 3, ('Wontons', 4.00): 3,
+         ('Crab Dip', 4.00): 3, ('Sliders', 4.00): 3,
+         ('Clams', 8.00): 3},
+         'Entrees': {('Salmon', 5.00): 3, ('Steak', 6.00): 3,
+                     ('Meat Tornado', 4.00): 3, ('A Literal Garden', 3.00): 3,
+                     ('Pasta', 4.00): 3, ('Ribs', 4.00): 3,
+                     ('Cabbage Rolls', 8.00): 3, ('Pizza', 4.00): 3,
+                     ('Paella', 4.00): 3},
+         'Desserts': {('Ice Cream', 500.0): 3, ('Cake', 30.00): 3,
+                      ('Pie', 3.00): 3, ('Pudding', 4.00): 3,
+                      ('Fruit', 4.00): 3, ('Sorbet', 4.00): 3,
+                      ('Torte', 4.00): 3, ('Flan', 4.00): 3,
+                      ('Apple', 4.00): 3},
+         'Drinks': {('Coffee', 4.00): 3, ('Tea', 3.00): 3,
+                    ('Blood of the Innocent', 6.00): 3, ('Pop', 4.00): 3,
+                    ('Wine', 4.00): 3, ('Beer', 4.00): 3,
+                    ('Sake', 4.00): 3, ('Cocoa', 4.00): 3,
+                    ('Evian', 4.00): 3},
+         'Sides': {('Frys', 4.00): 3, ('Salad', 3.00): 3,
+                   ('Bread', 6.00): 3, ('Slaw', 4.00): 3, ('Soup', 4.00): 3,
+                   ('Rice', 4.00): 3, ('Spinach', 4.00): 3,
+                   ('Sauce', 4.00): 3, ('Crab dip', 4.00): 3}}
+
+
+class Order:
+    """Class for Order"""
+    def __init__(self):
+        ''' Initialize with uuid ID and empty dict cart'''
```

Comment:
It is convention to always use double quotes for docstrings and single quotes for multiline strings.